.github/workflows/code_analysis.yaml

```diff
@@ -52,6 +52,10 @@ jobs:
                         name: 'Finalize classes'
                         run: vendor/bin/swiss-knife finalize-classes src tests --dry-run
 
+                    -
+                        name: 'Check before/after test fixture on no-changes'
+                        run: php bin/check-before-after-same-fixtures.php
```

Comment:
@samsonasik This might help with reviews 👍

Reply:
Ok, I removed at PR:

- https://github.com/rectorphp/rector-src/pull/7137